<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyController</class>
 <widget class="QWidget" name="MyController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>482</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>344</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>90</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>                 Отрезок                 Спектр отрезков</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>хн</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>yн</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="xstartEdit">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>30</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ystartEdit">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>60</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="xendEdit">
       <property name="geometry">
        <rect>
         <x>123</x>
         <y>30</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>253</x>
         <y>29</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Δθ</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>97</x>
         <y>60</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>yк</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>97</x>
         <y>30</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>хк</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="yendEdit">
       <property name="geometry">
        <rect>
         <x>124</x>
         <y>60</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>0</y>
         <width>20</width>
         <height>91</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="dEdit">
       <property name="geometry">
        <rect>
         <x>235</x>
         <y>54</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="dwawButton">
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Нарисовать</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>130</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Параметры отрисовщика</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>26</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Размер пикселя</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>53</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Цвет отрезка</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>199</x>
         <y>80</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>54</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Цвет фона</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="sizeEdit">
       <property name="geometry">
        <rect>
         <x>167</x>
         <y>26</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="foncolorButton">
       <property name="geometry">
        <rect>
         <x>229</x>
         <y>80</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Задать</string>
       </property>
      </widget>
      <widget class="QPushButton" name="linecolorButton">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Задать</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="fonButton">
       <property name="geometry">
        <rect>
         <x>37</x>
         <y>107</y>
         <width>100</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Цвет фона</string>
       </property>
      </widget>
      <zorder>label_5</zorder>
      <zorder>label_12</zorder>
      <zorder>label_13</zorder>
      <zorder>label_6</zorder>
      <zorder>label_15</zorder>
      <zorder>sizeEdit</zorder>
      <zorder>groupBox</zorder>
      <zorder>foncolorButton</zorder>
      <zorder>linecolorButton</zorder>
      <zorder>fonButton</zorder>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Выбор алгоритма</string>
      </property>
      <widget class="QRadioButton" name="cdaButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ЦДА</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="brDoubleButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Брезенхем (действ)</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="brIntButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Брезенхем (целые)</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="brModButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>311</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Брезенхем с устранением ступенчатости</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="standartButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Стандартный</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
